src/Bonsai.ML.Torch/ToArray.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Xml.Serialization;
+using System.Linq.Expressions;
+using System.Reflection;
+using Bonsai.Expressions;
+using static TorchSharp.torch;
+
+namespace Bonsai.ML.Torch
+{
+    /// <summary>
+    /// Converts the input tensor into an array of the specified element type.
+    /// </summary>
+    [Combinator]
+    [Description("Converts the input tensor into an array of the specified element type.")]
+    [WorkflowElementCategory(ElementCategory.Transform)]
+    [XmlInclude(typeof(TypeMapping<byte>))]
+    [XmlInclude(typeof(TypeMapping<sbyte>))]
+    [XmlInclude(typeof(TypeMapping<short>))]
+    [XmlInclude(typeof(TypeMapping<int>))]
+    [XmlInclude(typeof(TypeMapping<long>))]
+    [XmlInclude(typeof(TypeMapping<float>))]
+    [XmlInclude(typeof(TypeMapping<double>))]
+    [XmlInclude(typeof(TypeMapping<bool>))]
+    public class ToArray : SingleArgumentExpressionBuilder
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ToArray"/> class.
+        /// </summary>
+        public ToArray()
+        {
+            Type = new TypeMapping<double>();
+        }
+
+        /// <summary>
+        /// Gets or sets the type mapping used to convert the input tensor into an array.
+        /// </summary>
+        [Description("Gets or sets the type mapping used to convert the input tensor into an array.")]
+        public TypeMapping Type { get; set; }
```

Comment:
Why not just use an enum for this? I think I saw a compatible one elsewhere in the code.